modelservers/bert_squad/predictor.py

```diff
@@ -0,0 +1,20 @@
+from spell.serving import BasePredictor
+
+from transformers import pipeline
+from transformers.modeling_auto import AutoModelForQuestionAnswering
+from transformers.tokenization_auto import AutoTokenizer
+
+
+class Predictor(BasePredictor):
+    def __init__(self, config):
+        model = AutoModelForQuestionAnswering.from_pretrained("/model/model")
+        tokenizer = AutoTokenizer.from_pretrained("/model/tokenizer")
+        self.predictor = pipeline("question-answering", model=model, tokenizer=tokenizer)
+        with open("/code/context.txt") as f:
```

Comment:
I think this name is mismatched? Should we switch this to use a mount now? Or I guess this is pretty easy